packages/semantic-ui/package.json

```diff
@@ -28,7 +28,7 @@
     "test": "cross-env NODE_ENV=test BABEL_ENV=test jest"
   },
   "peerDependencies": {
-    "@rjsf/core": "^2.0.0",
+    "@rjsf/core": "2.4.2",
```

Comment:
Was this intentional? I think this code is still compatible with @rjsf/core 2.0.0-2.4.2?

Reply:
it was to make sure I am on the latest core.